src/test/java/unrc/dose/UserTest.java

```diff
@@ -92,4 +92,75 @@ public void userSet() {
 		assertEquals(name , pepe.get("username"));
 	}
 
-}
\ No newline at end of file
+	/**
+	 * delete an user
+	 * @result delete logically a user logged
+	 */
+	@Test
+	public void deleteUserSuccessfully(){
+		String username = "JohnConnor";
+		String password = "VeryHardPass";
+
+		assertEquals(true, User.deleteUser(username, password));
+	}
+
+	/**
+	 * delete Unsuccessfully user
+	 * @result user not found
+	 */
+	@Test
+	public void deleteUserUnsuccessfully(){
+		String username = "Pity";
+		String password = "Martinez";
+
+		assertEquals(false, User.deleteUser(username, password));
+	}
+
+	/**
+	 * Unsuccessfully basic login the user
+	 * @result user not found
+	 */
+	@Test
+	public void userLoginBasicUnsuccessfully(){
+		String username = "Pity";
+		String password = "Martinez";
+
+		assertEquals(false, User.login(username, password));
+	}
+
+	/**
+	 * Successfully basic login the user with data correct
+	 * @result user found whit data get into
+	 */
+	@Test
+	public void userLoginBasicSuccessfully(){
+		String username = "JohnConnor";
+		String password = "VeryHardPass";
+
+		assertEquals(true, User.login(username, password));
+	}
+
+	/**
+	 * Successfully basic login the user with password incorrect
+	 * @result user not found
+	 */
+	@Test
+	public void userLoginBasicUnsuccessfullyWithoutPassCorrect(){
+		String username = "JohnConnor";
+		String password = "VeryH";
+
+		assertEquals(false, User.login(username, password));
+	}
+
+	/**
+	 * Successfully basic login the user with password or username incorrect
+	 * @result user not found
```

Comment:
Please take care about the test description, because it doesn't really explain what does this test does.
Tests doesn't return anything, so the `@return` tag shouldn't be used.